### PR TITLE
Bug 1831366: upi/metal fix compact clusters after porting to terraform v0.12

### PR DESCRIPTION
**Pull request by varungur**

### CODE REVIEW

File: upi/metal/main.tf [hunk 1]
```diff
@@ -185,7 +185,7 @@ resource "aws_route53_record" "ctrlp_int" {
 }
 
 resource "aws_route53_record" "apps_noworker" {
-  count = var.worker_count == 0 ? 1 : 0
+  count = var.worker_count < 1 ? 1 : 0
```
Comment (abhinavdahiya) [inline upi/metal/main.tf:188]: does `== 0` not work anymore?

Comment (varungur) [reply upi/metal/main.tf:188]: After we moved to terraform version 0.12, the earlier fix stopped working.

**Comment by sdodson** [comment]
/retitle Bug 1831366: upi/metal fix compact clusters after porting to terraform v0.12

File: upi/metal/main.tf (hunk 1 above)
Comment (sdodson) [reply upi/metal/main.tf:188]: `== "0"` would probably have worked but this solution seemed more in line with apps_worker resource below

**Pull request merged**